packages/cache-utils/src/main.ts

```diff
@@ -37,7 +37,7 @@ const saveOne = async function (
     return true
   }
 
-  await del(cachePath, { force: true })
+  await rm(cachePath, { force: true, recursive: true, maxRetries: 3 })
```

Comment:
In the future. it might be nice to abstract these settings into a common place, so that all our IO ops use the same retry strategy?